pom.xml

```diff
@@ -141,7 +140,7 @@
 						<image>openjdk:14</image>
 					</from>
 					<to>
-						<image>${image.prefix}${project.artifactId}</image>
+						<image>registry.hub.docker.com/${docker.image.prefix}/${project.artifactId}</image>
```

Comment:
You might be able to revert this change since you ended up setting `jib.to.image` directly.